lib/cisco_node_utils/bgp_af.rb

```diff
@@ -273,6 +273,66 @@ def dampening
       val
     end
 
+    # Return true if dampening is enabled, else false.
+    def dampening_state
+      !dampening.nil?
+    end
+
+    # For all of the following dampening getters, haf_time, reuse_time,
+    # suppress_time, and max_suppress_time, return nil if dampening
+    # is not configured, but also return nil if a dampening routemap
+    # is configured because they are mutually exclusive.
+    def dampening_half_time
+      return nil if dampening.nil? || dampening_routemap_configured?
```

Comment:
Is there a case where dampening is not nil but dampening_routemap_configured is set?

In other words, if they're mutually exclusive, doesn't `dampening_routemap_configured` imply `dampening.nil?`